### PR TITLE
Capstone requirements

### PR DESCRIPTION
**Pull request by jlvFlores**
### Changes made on day 1:
- Added linters
- Updated README
- Added home and about HTML files
- Added CSS styles

### Changes made on day 2:
- Made the pages responsive
- Added JS file with a dropdown menu function and a dynamic HTML function
- Updated desktop version
- Updated styles
- Fix linter errors
- Updated README

### Changes made on day 3:
- Updated desktop page styles
- Created a [video describing my project](https://youtu.be/FNU_4C8fYSM) and added its link to the README.md file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          </li>
+          <li>
+            <a href="#">Join</a>
+          </li>
+          <li>
+            <a href="#">Sponsor</a>
+          </li>
+          <li>
+            <a href="#">News</a>
+          </li>
```
Comment (algerina) [inline index.html]: - [x] Kindly add the second navbar as per the template.
Template:
design

Your App:
yours

File: style.css [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+  color: #272a31;
+}
+
+.event-details a {
+  font-size: 1rem;
+  font-weight: 700;
+}
+
+.menu a:active {
+  color: #ec5242;
```
Comment (algerina) [inline style.css]: - [x] The hamburger menu background is supposed to be transparent or with the same color/pattern as the top section background.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      </div>
+      <div class="program-card">
+        <img class="svg-white" src="assets/images/Workshop.svg" alt="">
+        <h3 class="orange">Preparations</h3>
+        <p class="white">You can join virtually or physically, just know that the space is limited.</p>
+      </div>
+      <div class="program-card">
+        <img class="svg-white" src="assets/images/Network.svg" alt="">
+        <h3 class="orange">Projects</h3>
+        <p class="white">If you're interested in my works, feel free to explore the site.</p>
```
Comment (algerina) [inline index.html]: - [x] `alt` attributes are important in an image, the alt attribute provides alternative information for an image if a user for some reason cannot view it. Kindly add a description for all the images in your alt attribute.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @jlvFlores  ,

Good job so far! But you have some minor issues that need to be fixed.

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:
- No linters errors:heavy_check_mark:




## STATUS: CHANGED REQUIRED

- [ ] Kindly check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @jlvFlores, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**